---
title: "Pena Review"
output: md_document
fig_width: 4 
fig_height: 6 
---

# A Preprint review: Early life stress alters transcriptomic patterning across reward circuitry in male and female mice, CJ Penña et al. 2019 

## why review a preprint?

I saw [this paper](https://www.biorxiv.org/content/10.1101/624353v1) on Twitter and decided to take a closer look. One of my (unpublished) thesis chapters explores an RNA-seq dataset with four behavioral treatment groups and 3 brain regions. My postdoc research also uses datasets with mutliple beahvioral groups, 3 brain regions, and does so in males and females. So, while I may not have a publication record that refelts my expertise in this area, I'm quite familiar with the challenges of complex RNA-seq designs for beahivoral neuroscience research. I've read and presented a lot of preprints in journal club, but I've enver formally reviewed one, so I thought I would let this one be my first attempt! 

## The Iintroduction

Before I start reviewing a paper, I always consult [PLOS's how to review a paper video](https://reviewers.plos.org/resources/how-to-review-a-manuscript/). PLOS suggests that I start by reading the introdcution. I use read Adobe Acrobat Pro's read out loud feature to listen to the introduction while I make spegetti. This worked quite well! It was easy to follow and understand. 

## Data availablility

Without reading the methods or results, I went looking for public access to the data; I searched for Gene Expression Omnibus (GEO) accession numbers and found two:

* [GSE89757](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89757) is associated with  [Smith _et al._ 2016](http://www.eneuro.org/content/3/6/ENEURO.0240-16.2016) "Integrative Analysis of Disease Signatures Shows Inflammation Disrupts Juvenile Experience-Dependent Cortical Plasticity". This particular data set is used for the analysis associated with Fig. 6. 

* [GSE89692](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89692) is associated with [Penña _at el._ 2017](https://science.sciencemag.org/content/356/6343/1185.long) "Early life stress confers lifelong stress susceptibility in mice via ventral tegmental area OTX2". The published _Science_ paper focuses on **males only**, but the data for **both females and females** are archieved here.

Next, I went to [BioJupies](https://amp.pharm.mssm.edu/biojupies/) to see if I could quickly get a report of differentially gene expression. It didn't work on the first data set for some reason, but it did work for [GSE89692](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89692). BioJupies only lets you compare two groups: control and perterbed. The GEO accession has nearly 200 samples, but only about a dozen are imported into BioJupies. I can't tell what's what sex or what brain region the samples are from, but I can see the 4 treatment groups I'm limited to a single comparison in BioJupies, so I compare ELS-ctl and ELS-STVS to look at the effect of adult stressors in all mice that were subjected to early life stressors. [You can view the notebook here](https://amp.pharm.mssm.edu/biojupies/notebook/Ptbvkbvay).

I would like to have more data. I see that there are three supplemental tables, but those are rarely submitted to BioRxiv. The inputs and outputs to DESeq would be useful for reproducing the results. Is the beahivoral data publically available? There is no mention of a repository. 

## figures

**Figure 1:** Overall its a very clear, readable, interpretable figure. I like use of white/dark, no-stripes/stripes for showing the 2x2 design comparing control/stress and early-/late-life. I also like how this paper uses pink as the color to distinguish it from the blue color used in the 2017 study on males only. (Okay, I don't love the use of blue/pink as gender sterotypes, but it does effectively communicate sex differences :woman_shrugging:.) I do like the stats indicators and how subplots j and l provide a quick comparison of males and females. I do wonder why y'all chose to use bar plots with point instead of boxplots with or without points....  

**Figure 2:** This figure is very complex. It's the one that I wanted to [read on my phone](https://twitter.com/raynamharris/status/1124463113430913025), but the text was too small. Without zooming in, only the top portions with the venn diagrams and heatmaps are readable. The tables and hypergeometric plots are not readable. Also, none of the color-gradient legends are readbale. Even though I applaud the aurthors attempts to show all the data in various ways in one figure, I think a little less quantity would improve the quality. So, let me report on each part of figure 2. 


**Figure 2 venn diagrams:** At first glance, I like the venn diagrams because they do a nice job of conveying shared responses to stressors treatment. However, on closer inspection I noticed that the scaling is not consistent across all 6 venn diagrams, so you can't rely on size of the circle to compare across sexes or brain regions. [UpSet plots](https://github.com/hms-dbmi/UpSetR) that show bar graphs of intersectional data are increasingly popular; however they are easily to manipuate in terms of font size and bar color. I have been experimenting with bar plots to show the same data. The second image below show a bar graph version of the same data, transcripted from the text within the figure to the best of my ability. Colors for the bars were chosen by uploaded screen shots of the male and female venns and generating hex codes from <https://imagecolorpicker.com/>. I use "AS" to stands "adult stress" which encompaseses "Defeat" and "STVS".

![](fig2venn-original.png)


```{r setup, message = F, echo = F}
library(ggplot2)
library(dplyr)
library(cowplot)

knitr::opts_chunk$set(fig.path = './', echo = F, message = F)


# colors from imagecolorpicker.com

malecolors <- c("All" = "#79BDDE",  "AS vs ELS + AS" = "#A7C8D9", "ELS vs ELS + AS" = "#03BEEE" ,
                "ELS vs AS" = "#8DD6F2", "ELS + AS" = "#92C9DE", "AS" = "#DDDEE0", "ELS" = "#69D7FC" )

femalecolors <- c("All" = "#DE87B3", "AS vs ELS + AS" = "#D4AEC9", "ELS vs ELS + AS" = "#FF469F" ,
                 "ELS vs AS" = "#E2A1BF", "ELS + AS" = "#F3A4D5", "AS" = "#DDDEE0", "ELS" = "#FF80BB" )

sharedunshared <- c("All" = "#810f7c",  "AS vs ELS + AS" = "#8c6bb1", "ELS vs ELS + AS" = "#8c96c6" ,
                "ELS vs AS" = "#9ebcda", "ELS + AS" = "#fdb863", "AS" = "#e08214", "ELS" = "#b35806" )


```

```{r fig2venn-alt1}
fig2venn <- read.csv("fig2venn.csv")

#head(fig2venn)

#summary(fig2venn)

fig2venn$region <- factor(fig2venn$region, levels = c("VTA", "NAc", "PFC")) 
fig2venn$group <- factor(fig2venn$group, levels = c("All" ,  
                                                     "AS vs ELS + AS",
                                                     "ELS vs ELS + AS" ,
                                                     "ELS vs AS",
                                                     "ELS + AS",
                                                     "AS",
                                                     "ELS" 
                                                     ))


males <- fig2venn %>% filter(sex == "male") %>% droplevels()
females <- fig2venn %>% filter(sex == "female") %>% droplevels()

m <- ggplot(data=males, aes(x=group, y = number,  fill = group)) + 
  geom_bar(stat="identity")  + 
  coord_flip() + 
  facet_wrap(~region, nrow = 3) +
  scale_x_discrete(drop=FALSE)+
  scale_y_continuous(limits=c(0, 325)) +
  scale_fill_manual(values = malecolors) +
  geom_text(aes(label=number),size=3, hjust = -0.05) +
  guides(fill = guide_legend(reverse = TRUE)) +
  theme(legend.title = element_blank(),
        legend.position = "none",
        strip.background = element_rect( fill="white")) +
  labs(subtitle = "Male response",
       x = NULL, y = "number of DEGs") 
#m

f <- ggplot(data=females, aes(x=group, y = number,  fill = group)) + 
  geom_bar(stat="identity")  + 
  coord_flip() + 
  facet_wrap(~region, nrow = 3) +
  scale_x_discrete(drop=FALSE)+
  scale_y_continuous(limits=c(0, 325)) +
  scale_fill_manual(values = femalecolors) +
  geom_text(aes(label=number),size=3, hjust = -0.05) +
  guides(fill = guide_legend(reverse = TRUE)) +
  theme(legend.title = element_blank(),
        legend.position = "none",
        strip.background = element_rect( fill="white")) +
  labs(subtitle = "Female response",
       x = NULL, y = "number of DEGs") 
#f

plot_grid(m, f + theme(axis.text.y=element_blank()), 
          nrow = 1, rel_widths = c(0.6,0.4))
```

I like the above bar plot because it lets me compare DEGs across sexs and brain regions. I see the largest responses in the VTA of males and the NAc of females. The PFC is most responsive to early and late stress combined, with little overlap to the other treatments. But, what do the results say when referencing the venn diagrams?

_There was 1-23% overlap
in DEGs across the three stress conditions (ELS or
STVS/Defeat or ELS+STVS/Defeat) in male and
female VTA, male NAc, and male and female PFC
(Figure 2a, f, i, q, v), and more than 50% overlap in
female NAc, compared to a standard-reared control
with no adult stress (Std-Ctl; Figure 2n)._

Hmm. This doesn't say another about, but quotes thresholds for percent overlap. I'm not really sure what they mean by overlap. I _assume_ it means the shaded regions of the venn diagrams, but I am not certain. To confirm results, I had to caluculate percentage of overlap rather than plot total number of genes. I made a color palete where orange shades represent unique and purples reprsented shared (aka overlappping) gene expression responses. First I calculate the percent as a function of the sum total of DEGs. This does not relate to the results, but I do think it provides a nice alternative to venn diagrams. After than, I calculated percent of DEGs for each group and then plotted the percentage, adding lines for 23 and 50%, which were noted in the manuscript. I don't think this figure aggrees with the stated results. I see the female VTA and NAc and the male PFC as having greater than 25% DEGs shared between one or more stress groups, which the other three show < 25%. I do see that in the female NAc, 50% of the genes are unique to either ELS or AS alone. Maybe this is what those results refer to...

```{r fig2venn-alt2}
fig2venn$percent <- (fig2venn$number / sum(fig2venn$number) * 100)

percentbytotal <- ggplot(fig2venn, aes(x = region, y = percent, fill = reorder(group, desc(group)))) + 
  geom_bar(stat = "identity") + 
  theme(legend.title = element_blank()) + 
  labs(x = NULL, y = "% DEGs \n of total",
       caption = "Another alternative to Venn diagrams.") +
  geom_hline(yintercept=23, linetype="dashed") +
  facet_wrap(~sex) +
  scale_fill_manual(values = sharedunshared) 


subsetpercent <- function(whichsex, whichregion){
  newdf <- fig2venn %>% filter(sex == whichsex, region == whichregion)%>% droplevels()
  newdf$percent <- (newdf$number / sum(newdf$number) * 100)
  return(newdf)
}

MVTA <- subsetpercent("male", "VTA")
MNAc <- subsetpercent("male", "NAc")
MPFC <- subsetpercent("male", "PFC")
FVTA <- subsetpercent("female", "VTA")
FNAc <- subsetpercent("female", "NAc")
FPFC <- subsetpercent("female", "PFC")

fig2venn <- rbind(MVTA, MNAc,MPFC,FVTA, FNAc, FPFC)


fig2venn$sex <- factor(fig2venn$sex, levels = c("female", "male"))

percentbygroup <- ggplot(fig2venn, aes(x = region, y = percent, fill = reorder(group, desc(group)))) + 
  geom_bar(stat = "identity") + 
  theme(legend.title = element_blank()) + 
  labs(x = NULL, y = "% DEGs \n by group",
       caption = "This graph better matches the stated results.") +
  geom_hline(yintercept=22, linetype="dashed") +
  geom_hline(yintercept=50, linetype="dashed") +
  facet_wrap(~sex) +
  scale_fill_manual(values = sharedunshared) 


plot_grid(percentbytotal, percentbygroup, nrow = 2)
```

**Figure 2 heat maps:** I don't often see two heatmaps with different clustering approaches shown side by side, but I like this. I'm more used to seeing the one's on the bottom that are clustered by correlation, but I like the addtional infromation gained by the "union" type. However, I can't help but wonder a) why the authors chose to show group averages rather than showing the data for all the samples. Also, why don't the authors provide a heatmap that shows how all the samples compare to one another? Do samples cluster mostly by brain region, sex, early-life stress or late-life stress? These are questions that could be easily deomonstrated with a supplementary heatmap or PCA plot or MDS plot in the supplemental material.  



**Figure 2 tables:** Figure 2 has some tables-like subplots. I combine them for easy viewing and immediately wonder are the female tables shaded blue not pink. From this data, the authors infer that neither ELS, adult stress, or a combination of stresses induced cell type-specific transcriptional changes. From what they present, I concur. I like seeing the author's use cell-type specific markers to validate or explore cell-specific patterns. A revierwed ask me to do this recently, and I think really improved that manuscript (see [the preprint](https://www.biorxiv.org/content/10.1101/153585v3)). My only concern with this is that I was provided any of the raw data in the methods or figures, so all I have to go on are reported p-values and odds ratios. What genes were used as markers? We they all expressed? How many were significant?

![](fig2table-1.png)

**Figure 2 hypergeometric plots:** These are really cool and agree with the results. I do think is easier to compare the results to the figure if you position them closer in space. Like this. Also, the figure legend does a great job of explaining each quadrant, but I find that the exes within the figures are hard to read.  

![](fig2geo.png)


## conflicts of interest

I've heard of the the [Eric Nestler](https://en.wikipedia.org/wiki/Eric_J._Nestler), but I don't personally know any other the authors. I don't share any co-authors with Catherine Peña, but we both do follow 38 of the same people on Twitter